cmd/juno-cli/cli/root.go

```diff
@@ -141,6 +144,6 @@ func initClient() *feeder.Client {
 // Execute handle flags for Cobra execution.
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		Logger.With("Error", err).Error("Failed to execute CLI.")
+		Logger.Fatal("Failed to execute Juno.")
```

Comment:
Since Juno is somewhat monolithic, I think it pays to be specific about which part failed to run, here the CLI.